### ISSUE TITLE
The file cannot be written, and an error occurred.

### ISSUE THREAD

**Issue by zhouyujn** [microsoft/Document-Knowledge-Mining-Solution-Accelerator#15]
After the deployment is complete, I can log in to the web page, but file uploads fail with the following error:

Send-FilesToEndpoint : Failed to upload file: Accessibility-in-Housing-Report.pdf. Status code: ServiceUnavailable                                           所在位置 C:\Document-Knowledge-Mining-Solution-Accelerator-main\Deployment\uploadfiles.ps1:11 字符: 5                             +     Send-FilesToEndpoint -DataFolderPath "..\Data" -EndpointUrl "${En ...     

Check the blob storage, I do not see the container in the storage, and the corresponding service principal does not have blob-related permissions. The documentation does not provide detailed descriptions. I would like to ask how to resolve this.

**Comment by Prasannakatakam-Microsoft** [comment]
@zhouyujn : Will work the issue and will update the progress.

**Comment by Vinay-Microsoft** [comment]
@zhouyujn can you please share the files or give some information about the files which you were trying to upload?

**Comment by Prasanjeet-Microsoft** [comment]
Hello @zhouyujn, Could you please provide the complete error details? It appears that the information is being cut off, and we are unable to see the full error message. Your assistance in sharing the entire error message would be greatly appreciated and would help us better understand the issue.

**Comment by zhouyujn** [comment]
Thank you very much for everyone's replies. I reinstalled it last night, and it seems that the error is no longer there.

**Comment by Prasanjeet-Microsoft** [comment]
> Thank you very much for everyone's replies. I reinstalled it last night, and it seems that the error is no longer there.

@zhouyujn Thank you for the update. Since you're no longer facing the error, I am closing this issue. If you encounter any other problems, please feel free to open a new issue. 

Happy to help!
